local/build-binary

```diff
@@ -133,6 +133,9 @@ if [[ "${WITH_TOKUDB}" = "ON" ]]; then
     if [[ "${ANALYZER_OPTS}" = *WITH_VALGRIND=ON* ]]; then
         CMAKE_OPTS+=" -DUSE_VALGRIND=ON"
     fi
+else
+    # temporary add workaround for PS-4742
```

Comment:
Could be no longer required, 8.0 now has both TokuDB and Tokubackup

Reply:
fixed